### ISSUE TITLE
is that IP ?

### ISSUE THREAD

**Issue by notevil1337** [UnamSanctam/SilentCryptoMiner#1885]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/6a7762ee-f279-4d1d-949c-38f74e70cc41)
Hello unam, is that ip or what?

**Comment by notevil1337** [comment]
Also they are mining for like 3 hours and on the pool they didnt appear..
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141959915/49455fae-220d-4d4f-b723-97fbef0bc3cd)

**Comment by UnamSanctam** [comment]
Yes that's an IPv6 address. The 21 MH/s should find one relatively quickly, do you not see that ones worker name in 2miners? Also make sure that you're using port 16060 (SSL) or port 6060 (non-SSL) and not accidentally port 16161 or 6161.

**Comment by notevil1337** [comment]
yeah im using 16161 port, and its mining well now, it got fixed
